### PR TITLE
make coach tabs more consistent

### LINKED ISSUE #2577: Consistent display of empty coach tabs
### Observed behavior

As noted by @MauHernandez in #2576, each tab in the coach reports has its own way of displaying that there is no relevant data to display. This should be standardized.

### Expected behavior

Each empty descriptor should have the same styling and page positioning, in addition, the use of terminal punctuation across each should be consistent.

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

Continues the work of @MauHernandez

* Removes `report-subheading` because it had too much conditional page-rendering logic and no styling in it. Instead, let individual pages control their own subheader as a simple `<p>`.
* Fixed up conditional logic for allowing 'new' buttons to be left-aligned when page is empty
* Consistent display of `<h1>` in all pages

### Before

image

image

image

image

image

### After

image

image

image

image

image

### Reviewer guidance

Please test this in _non-empty_ states

### References

supersedes #2653 to fix #2577

----

### Contributor Checklist

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] Contributor has fully tested the PR manually
- [x] Screenshots of any front-end changes are in the PR description
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by benjaoming** [comment]
@indirectlylit something fishy with the commit history, as if it's based off the wrong branch?

**Comment by indirectlylit** [comment]
I noticed those weird merge commits also, but as far as I can tell the changes are exactly what they should be: https://github.com/learningequality/kolibri/pull/3084/files

**Comment by indirectlylit** [comment]
Tried a couple ways to get rid of the merge commits, but couldn't figure anything out that wasn't going to take a bunch of manual work or lose commit history.

File: kolibri/plugins/coach/assets/src/views/reports/channel-list-page.vue [hunk 1]
```diff
@@ -62,17 +65,20 @@
...
       activityCell,
     },
     mixins: [alignMixin],
     $trs: {
       recentTitle: 'Recent activity',
+      topicsTitle: 'Content',
       channels: 'Channels',
       channelList: 'Channel list',
       lastActivity: 'Last active',
+      showingRecent: 'Showing activity in past {threshold} days',
```
Comment (jonboiser) [inline kolibri/plugins/coach/assets/src/views/reports/channel-list-page.vue:79]: Will the `threshold` number be localized correctly this way?

Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/reports/channel-list-page.vue:79]: I .. think so. But will check

**Comment by jonboiser** [comment]
I updated the broken test here in a PR https://github.com/indirectlylit/kolibri/pull/23

File: kolibri/plugins/coach/assets/src/views/reports/channel-list-page.vue (hunk 1 above)
Comment (jonboiser) [reply kolibri/plugins/coach/assets/src/views/reports/channel-list-page.vue:79]: These are new strings, so there's no way to check manually right now. Just not sure if you need some extra configuration to make sure it doesn't use arabic numerals by default.

**Comment by indirectlylit** [comment]
ongoing task related to numeral localization: #2788

**Pull request merged**